keyboards/preonic/keymaps/juno/keymap.c

```diff
@@ -0,0 +1,314 @@
+/* Juno's Preonic Layout  */
+
+#include "preonic.h"
+#include "action_layer.h"
```

Comment:
Could you remove all of these additional includes?  

```suggestion
```

They get included automatically, at various points in the built process, so they are redundant and not necessary here.